src/styles/App.css

```diff
@@ -1,5 +1,17 @@
-.App {
+.weather-app {
   text-align: center;
+  margin: 0 20px;
+}
+
+.location {
+  text-align: left;
+  margin: 0 0px;
+}
+
+.search-form {
+  text-align: left;
+  margin: 0 0px;
+  margin-bottom: 30px;
```

Comment:
All the classes below this are hangovers from the initial installation, they arent being used any more so should be deleted